en/health/centreon-agent.md

```diff
@@ -0,0 +1,473 @@
+    ```
+
+    Example:
+    
+    ```yaml
+    /usr/sbin/centreon-agent config \
+    --token aaaa-aaaa-aaaa-aaaa \
+    --type poller \
+    --output /etc/centreon-agent/centreon-agent.yml
+    ```
+
+3. Add an **environment** [tag](#tags):
+
+    Open the `centreon-agent.yml` file generated at installation (cf. `--output` option configured earlier) and add the following instructions under the **collect** section.
+
+    ```yaml
+    collect:
+    tags:
+        environment: [staging|preproduction|production|your-custom-value]
+    ```
+
+    Example:
+
+    ```yaml
+    collect:
+    tags:
+        environment: production
+    ```
+
+    If you have multiple environnements of the same kind, you can _suffix your type of environnement, for instance: "production_client1".
+
+4. Enable the **centreon-agent** Service:
+
+    ```
+    systemctl enable centreon-agent.service
+    ```
+
+5. Start the **centreon-agent** service:
+
+    ```
+    systemctl start centreon-agent.service
+    ```
+
+7. You can now [configure your Agent](#configuring-the-agent) (Gateway, proxy etc.) and then [test](#testing-the-agent) your overall configuration.
+
+## Configuring the Agent
+
+### Network
+
+If an Agent does not have direct access to the outside, two options allow you to circumvent this: access through an HTTP proxy and/or access through the Gateway mode. In the latter, the Agent that needs access (called “Gateway Client”) can get through an other Agent (called “Gateway Server”) that does have access to the outside.
+
+**Example**
+
+Your infrastructure is protected within a closed system and you have a proxy Server to manage all outgoing traffic. The Agent installed on the machine hosting the Centreon Central Server is the only one you want to grant access to the outside. In this case, you could configure your network as such:
+
+- Configure the proxy option on the Central Agent to grant it access to the outside
+
+- Configure this same Agent as a Gateway Server
+
+- Configure all other Agents (installed next to Pollers, Remote Servers, MAP, etc.) as Gateway Clients
+
+#### Proxy Configuration
+
+If you have a proxy access configured on the host machine, copy the proxy settings to the `/etc/centreon-agent/centreon-agent.yml` file under **output**:
+
+```yaml
+output:
+  token: [your-token]
+  proxy_url: [your-proxy-address]:[your-desired-port]
+  proxy_ssl_insecure: [true|false]
+```
+
+Example:
+
+```yaml
+output:
+  token: aaaa-aaaa-aaaa-aaaa
+  proxy_url: http//proxy.local.net:3128
+  proxy_ssl_insecure: false
+```
+
+You then need to restart the Agent:
+
+```
+systemctl restart centreon-agent.service
+```
+
+#### Gateway Configuration
+
+- Gateway Server: copy the following code to the `/etc/centreon-agent/centreon-agent.yml` file of the Agent that will act as a Gateway server. You must add the token for the gateway (which is different from the one you entered when configuring the `centreon-agent.yml` file).
```

Comment:
Once again, no token is a "must".
Only if the user want to further secure communication between gateway agents that he **may** use **a** token of his